include/nanogui/theme.h

```diff
@@ -31,50 +31,83 @@ class NANOGUI_EXPORT Theme : public Object {
     Theme(NVGcontext *ctx);
 
     /* Fonts */
+    /* NOTE: updating documentation for Theme::GlobalDefaultFonts?  Please also
+     *       update the manual documentation in python/theme.cpp at the bottom.
+     *       This class is not processed by docs/mkdoc_rst.py.
+     */
     /**
-     * The ``"sans"`` font.  Some widgets allow for a user-specified font face
-     * (e.g., \ref nanogui::Label).  The default action is to use
-     * \ref Theme::defaultFont to set this, however in the event that a
-     * ``nanogui::Theme`` class is not available, this method is called instead.
-     * A ``nanogui::Theme`` class will not be available when no parent is
-     * specified.  For example,
-     * ``new nanogui::Label(nullptr, "Some Text")``.  Doing this is not
-     * explicitly forbidden, but is generally discouraged unless you really know
-     * what you are doing.
+     * \struct GlobalDefaultFonts theme.h nanogui/theme.h
+     *
+     * \brief The font face string identifiers that are always loaded / available
+     *        for every Widget.
+     *
+     * In the (rare) event that a Widget does not have a Theme instance (this only
+     * happens when a Widget is constructed without a parent), these values are also
+     * used in the Widget font getter methods.
      *
-     * Both this method and \ref defaultFont are ultimately responsible for
-     * setting both \ref Widget::mFont and \ref Widget::mTooltipFont.
-     */
-    static std::string globalDefaultFont() { return "sans"; }
-    /// Similar to \ref globalDefaultFont, only ``"sans-bold"`` is used.
-    static std::string globalDefaultBoldFont() { return "sans-bold"; }
-    /// Similar to \ref globalDefaultFont, only for icon fonts.
-    static std::string globalDefaultIconFont() { return "icons"; }
+     * \sa The *implementation* of \ref nanogui::Widget::font.
+     */
+    struct GlobalDefaultFonts {
```

Comment:
Oh...and this is the most ugly thing I've done in a while.  C++11 alternative would be `static constexpr auto Normal = "sans"` which I like very much, but I couldn't figure out how to bind that with pybind (I think because `constexpr` has no memory).  Is there a more modern alternative?